recipes/iterative-solver/meta.yaml

```
@@ -0,0 +1,67 @@
+{% set name = "iterative_solver" %}
+{% set version = "0.10.15" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/molpro/iterative-solver/archive/refs/tags//{{ version }}.tar.gz
+  sha256: 895cddbd1b9578b7111c546cd0d5c912add96bc9d268e6da26623c8ec5d59efe
+
+build:
+  skip: True  # [win]
+  script:
+    - export VERSION={{ version }}
+    - cmake -DCMAKE_INSTALL_PREFIX=$PREFIX  -DCMAKE_CXX_FLAGS=-fPIC  -DCMAKE_BUILD_TYPE=Release  -DDEPENDENCYMANAGER_FETCHCONTENT=OFF  -DLINEARALGEBRA_ARRAY_HDF5=OFF -DLINEARALGEBRA_ARRAY_GA=OFF  -DFORTRAN=OFF -DBUILD_SHARED_LIBS=ON -DLIBRARY_ONLY=ON -B cmake-build -S .
+    - cmake --build cmake-build --config Release -t install
+    - echo '__version__ = "'$VERSION'"' > python/iterative_solver/_version.py
+    - cd python && {{ PYTHON }} -m pip install . -vvv
+  number: 0
+
+requirements:
+  build:
+    - python     # [build_platform != target_platform]
+    - cross-python_{{ target_platform }}     # [build_platform != target_platform]
+    - cython >=3.0                                 # [build_platform != target_platform]
+    - {{ compiler('c') }}
+    - {{ stdlib("c") }}
+    - {{ compiler('cxx') }}
+    - cmake >=3.14
+    - make
+    - pip
+  host:
+    - python
+    - pip
+    - cython >=3.0
+    - numpy
+    - mpich
+    - eigen
+    - liblapacke
+    - libblas
+  run:
+    - python
+
+test:
+  imports:
+    - iterative_solver
+  requires:
+    - pip
+  commands:
+    - pip check
```

let's explicitly test for the presence and naming of the packages here. It's slightly redundant (presumably the python import would fail otherwise), but it makes it clearer what this recipe is actually packaging.
```suggestion
  commands:
    - pip check
    # these could eventually move into a separate library ouput
    - test -f $PREFIX/lib/libmolpro-iterative-solver${SHLIB_EXT}    # [unix]
    - test -f $PREFIX/lib/libmolpro-profiler${SHLIB_EXT}            # [unix]
    - test -f $PREFIX/lib/libmolpro-utilities${SHLIB_EXT}           # [unix]
```